mne/channels/channels.py

```diff
@@ -582,11 +582,10 @@ def anonymize(self):
         """
         anonymize_info(self.info)
         if hasattr(self, 'annotations'):
-            if self.annotations is None:
-                pass
-            else:
-                self.annotations.orig_time = None
-                self.annotations.onset -= self._first_time
+            # XXX : anonymize should rather subtract a random date
+            # rather than setting it to None
```

Comment:
Why is that? This seems fine

Reply:
that's what does mne_anonymize and what recommends the BIDS spec. One motivation is to change the date of a full study, eg to know in what is the order of the subjects and to know in which order the runs occurred. I wrote it down to not forget.

Reply:
IMHO, set it to None is the most natural way and allows to check for anonymity. But I also understand the value of setting an arbitrary date. Then we would need to have a flag or something to indicate that the date was already altered for anonymity.

Either way, we should open an issue.

Reply:
you should make the date before 1900 according to BIDS spec (if I remember correctly), so it's obvious it has been anonymized. The date is not completely arbitrary :)